sim/src/mpc.py

```diff
@@ -837,6 +857,15 @@ def CalcOutput(self, context, output):
     v0 = np.array(config['v0'])
     plant.SetPositions(plant_context, q0)
     plant.SetVelocities(plant_context, v0)
+        
+    # Change link masses
+    if config['model_error']['enabled']:
+        for body in plant.GetBodyIndices(plant.GetModelInstanceByName("achilles")):
```

Comment:
Looks like some bodies are small enough where (# Change link masses by m <-- m * r, ) becomes a problem. Either we are careful about the mu and sigma combo, or we remove the small mass links from being varied.

Reply:
good catch, added a check to enforce positive mass